package.json

```
@@ -1,18 +1,26 @@
 {
   "scripts": {
-    "prettier-write": "prettier --write '{app,frontend}/**/*.{js,css}'",
-    "prettier-check": "prettier --check '{app,frontend}/**/*.{js,css}'"
+    "build:css": "postcss ./app/assets/stylesheets/application.css -o ./app/assets/builds/application.css",
+    "build:js": "bin/esbuild",
```

To parallelize the two in one script:
```suggestion
    "build:css": "postcss ./app/assets/stylesheets/application.css -o ./app/assets/builds/application.css",
    "build:js": "bin/esbuild",
    "build": "run-p build:*",
```

With [npm-run-all](https://www.npmjs.com/package/npm-run-all). I guess it's OK to bundle the two as wer'e not going to debug these that often from within a docker container.